File: room.go
package shipyard

// Room represents a physical room containing AV Equipment
type Room struct {
	ID                 string `json:"id"`
	Designation        string `json:"designation"`
	PublicDescription  string `json:"publicDescription"`
	PrivateDescription string `json:"privateDescription"`
	ProxyBaseURL       string `json:"proxyBaseURL"`
	Tags               Tags   `json:"tags"`
}

File: ui-old/src/app/components/room-page/room.ts
export class Room {
    id: string;
    designation: string;
    publicDescription: string;
    privateDescription: string;
    proxyBaseURL: string;
    tags: Map<string, string>;

    constructor() {
        this.tags = new Map();
        this.id = "";
        this.designation = "";
        this.privateDescription = "";
        this.publicDescription = "";
        this.proxyBaseURL = "";
    }
}